x-pack/docs/en/ilm/index.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[index-lifecycle-management]]
+= Managing Indices
+
+:ilm:             index lifecycle management
+:ILM:             Index lifecycle management
+[partintro]
+--
+The {ILM} APIs enable you to automate how you want to manage your indices over
+time. Rather than simply performing management actions on your indices on a set
+schedule, you can base actions on other factors such as shard
+size and performance requirements.
+
+You control how time series indices are handled as they age by attaching a
+lifecycle policy to the index template used to create them. You can update
+the policy to modify the lifecycle of both new and existing indices.
+
+There are four stages in the index lifecycle:
```

Comment:
Since Index Life Cycle Management is a broader system beyond just the timeseries type with hot-warm-cold-delete, we should start off with introducing it in broader terms and then further sub-dividing it to mention that timeseries is just one instance.

It probably sounds like overkill since why generalize when we only have implemented one of the types (timeseries), but it will set the scene for expanding our discussion once we introduce more types that may have their own unique phases.

does that make sense?